en: support multiple sources create/stop (#292)
pingcap/docs-dm: en/manage-source.md

## Flags description

> + `create`: Create one or more upstream database sources. When creating multiple data sources fails, it rolls back to the state before executing the command.

Reviewer: ```suggestion
+ `create`: Creates one or more upstream database source(s). When creating multiple data sources fails, DM rolls back to the state where the command was not executed.
```

+ `update`: Updates an upstream database source.

> + `stop`: Stops one or more upstream database sources. When stopping multiple data sources fails, some data sources might been successfully stopped.

Reviewer: ```suggestion
+ `stop`: Stops one or more upstream database source(s). When stopping multiple data sources fails, some data sources might be stopped.
```
